wormhole-connect/src/routes/types.ts

```diff
@@ -41,14 +66,18 @@ export const isSignedCCTPMessage = (
   typeof message === 'object' &&
   'message' in message &&
   'attestation' in message;
+export const isSignedNttMessage = (
+  message: SignedMessage,
+): message is SignedNttMessage =>
+  isSignedWormholeMessage(message) && isUnsignedNttMessage(message);
 
 export interface TransferInfoBaseParams {
-  txData: ParsedMessage | ParsedRelayerMessage;
```

Comment:
is there a way to avoid adding this type for a single route (NTT) to this type which is used by all routes?

Reply:
Give that only the 'relayerFee` property is used in the `UnsignedNttMessage` and it's already present on the `ParsedRelayerMessage`, I think this type can be removed from the union. However `ParsedRelayerMessage` is currently specific to the `relay` route, so it's still kind of weird. I feel like this should be refactored so we don't have to rely on these union types.